lte/gateway/python/magma/subscriberdb/processor.py

```diff
@@ -324,3 +354,4 @@ def sqn_to_seq(cls, sqn):
             seq (int): the sequence number
         """
         return sqn >> 5
+        
```

Comment:
this blank line should not contain any whitespaces

Reply:
Comment addressed.